builtin/tfc/config_sourcer.go

```diff
@@ -54,7 +55,7 @@ type ConfigSourcer struct {
 }
 
 type cachedSecret struct {
-	Outputs map[string]string // The most recent outputs for a workspace
+	Outputs map[string]StateIncludeAttributes // The most recent outputs for a workspace
```

Comment:
Caching the full TFC attributes response now, because it's useful to look at the value and the type later on, and we're now allowing types other than `string.`